kolibri/plugins/base.py

```diff
@@ -37,6 +37,29 @@ class KolibriPluginBase(object):
     This is the base class that all Kolibri plugins need to implement.
     """
 
+    # Name of a local module that contains url_patterns that define
+    # URLs for views that do not contain any
+    # translated content, and hence will not be prefixed
+    # with a language prefix
+    untranslated_view_urls = None
+
+    # Name of a local module that contains url_patterns that define
+    # URLs for views that contain
+    # translated content, and hence will be prefixed
+    # with a language prefixs
+    translated_view_urls = None
+
+    # Name of a local module that contains additional settings to augment
+    # Django settings.
+    # For settings that take a tuple or list, these will be appended to the value from
+    # the base settings module set through conventional Django means.
+    django_settings = None
```

Comment:
You can use the `#: Comment` syntax to automatically have the comment become a documentation string for the property :) It's not a part of default Sphinx, but we have added it to make inline documentation easier.

Reply:
Does it still work?

Reply:
Oh neat! And yes, it does.